### DISCUSSION TITLE
Permissions for viewing not assigned task

### DISCUSSION THREAD

**Discussion by niels-przybilla** [kimai/Paid-plugins#162] [General]
Hello,

I have some users which should see only tasks which are assigned to them.
At the moment it seems that they can also see tasks which are not assigned to anyone. Even tasks which are assigned to another group.
E.G. Marketing team is seeing tasks for DEV team ... which is information overload.

Did I miss anything that i need to configure?

THX NIels

**Comment by kevinpapst** [comment]
Task that are not assigned to anyone are visible to anyone. The assumption is that those can be picked by everyone.

Tasks that are assigned to a team are visible to the team only.

Behavior changes depending on the configured permissions:
https://www.kimai.org/documentation/plugin-task-management.html#permissions

AFAIR you can configure it to work in three different ways
- only the “My tasks” widget will be shown, listing page is hidden from “normal users” and teamleads will plan the work schedule (a list of tasks) for the users, who then use the widget to start/pause/stop their assigned tasks
- allow self-management by the user via the dashboard widgets, by assigning to pending tasks
- tasks are shared, pending tasks can be started and stopped by everyone/team members
